ui/src/app/pages/workspace/workspace-library.tsx

```diff
@@ -86,6 +86,7 @@ export const WorkspaceLibrary = withUserProfile()
 
   async componentDidMount() {
     this.loadPublishedWorkspaces();
```

Comment:
This is kind of bad. This function is what populates the `workspaceList`, which is what we use to determine whether or not we want to display anything in the library at all.

I could rename the function to `loadWorkspaceList` because that is what it actually does. 

We should probably also call this to refresh the `workspaceList` whenever we switch tabs instead of just whenever we load the Published Workspaces tab. The Featured Workspaces tab just filters down the Published Workspaces list by what we have decided to feature.